package.json

```diff
@@ -16,7 +16,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/ajvwhite/homebridge-smartthings-routine-triggers.git"
+    "url": "git+https://github.com/needlerp/homebridge-smartthings-routines.git"
```

Comment:
For a pull request - homepage references and URLs need to remain as it - of course we should add you as a contributor to the readme, and your input is much appreciated